package.json

```diff
@@ -1,55 +1,85 @@
+    "lint:fmt": "prettier --check --log-level=warn .",
+    "lint:fmt:fix": "prettier --write .",
+    "lint:types": "tsc --noEmit --project tsconfig.json",
+    "prepare": "husky install",
+    "yata-fetch": "dotenv -- yata-fetch --config .yata.json",
+    "test": "npx playwright test",
+    "test:ui": "npx playwright test --ui",
+    "test:debug": "npx playwright test --debug"
   },
   "devDependencies": {
-    "@babel/core": "^7.18.13",
-    "@storybook/addon-actions": "^6.5.10",
-    "@storybook/addon-essentials": "^6.5.10",
-    "@storybook/addon-interactions": "^6.5.10",
-    "@storybook/addon-links": "^6.5.10",
-    "@storybook/addon-postcss": "^2.0.0",
-    "@storybook/builder-webpack5": "^6.5.10",
-    "@storybook/manager-webpack5": "^6.5.10",
-    "@storybook/react": "^6.5.10",
-    "@storybook/testing-library": "^0.0.13",
-    "@types/node": "18.6.5",
-    "@types/react": "18.0.17",
-    "@types/react-dom": "18.0.6",
-    "@typescript-eslint/eslint-plugin": "^5.33.0",
-    "@typescript-eslint/parser": "^5.34.0",
-    "autoprefixer": "^10.4.8",
-    "babel-loader": "^8.2.5",
-    "eslint": "8.21.0",
-    "eslint-config-next": "12.2.4",
-    "eslint-config-prettier": "^8.5.0",
-    "eslint-plugin-storybook": "^0.6.4",
-    "postcss": "^8.4.16",
-    "tailwindcss": "^3.1.8",
-    "typescript": "^4.7.4"
+    "@esbuild-plugins/node-globals-polyfill": "0.2.3",
+    "@lukso/web-components": "^1.28.0",
+    "@nuxt/devtools": "latest",
+    "@nuxtjs/device": "^3.1.0",
+    "@nuxtjs/plausible": "^0.2.1",
+    "@nuxtjs/tailwindcss": "^6.8.0",
+    "@pinia/nuxt": "^0.4.11",
+    "@playwright/test": "1.36.2",
+    "@types/fs-extra": "11.0.1",
+    "@types/node": "^18.17.0",
+    "@types/randombytes": "2.0.0",
+    "@typescript-eslint/eslint-plugin": "6.2.0",
+    "@typescript-eslint/parser": "6.2.0",
+    "buffer": "6.0.3",
+    "dotenv-cli": "7.2.1",
+    "eslint": "8.45.0",
+    "eslint-config-prettier": "8.9.0",
+    "eslint-import-resolver-typescript": "3.5.5",
+    "eslint-plugin-import": "2.28.0",
+    "eslint-plugin-json": "3.1.0",
+    "eslint-plugin-playwright": "0.15.3",
+    "eslint-plugin-testing-library": "5.11.0",
+    "eslint-plugin-vue": "9.15.1",
+    "fs-extra": "11.1.1",
+    "https-browserify": "1.0.0",
+    "husky": "8.0.3",
+    "lint-staged": "13.2.3",
+    "npm-run-all": "4.1.5",
+    "nuxt": "^3.6.5",
+    "postcss-html": "^1.5.0",
+    "prettier": "3.0.0",
+    "rollup-plugin-polyfill-node": "^0.12.0",
+    "sass": "1.64.1",
+    "stream-browserify": "3.0.0",
+    "stream-http": "3.2.0",
+    "stylelint": "15.10.2",
+    "stylelint-config-prettier": "9.0.5",
+    "stylelint-config-recommended-vue": "1.5.0",
+    "stylelint-config-standard-scss": "10.0.0",
+    "stylelint-prettier": "4.0.2",
+    "stylelint-scss": "5.0.1",
+    "typescript": "5.1.6",
+    "util": "0.12.5",
+    "yata-fetch": "2.1.5"
+  },
+  "lint-staged": {
+    "*.{ts,js,vue}": [
+      "prettier --list-different",
+      "eslint"
+    ],
```

Comment:
I'm not a big fan of this lint staged thing... But up to you.

I rly think a good CI and branch protection is enough.

Reply:
But lint staged is important otherwise you cannot partially commit things

Reply:
This allow to catch issues earlier then on CI, I guess you can still run it from script but this is nice small automation that helps you not forget. If you are not liking this I will remove.

Reply:
It is really up to you, if you think it adds more value than "blocking time" then lets keep it.